### PR TITLE
Js assignment 1, 2, 3, 4, 5, 6

### PR DESCRIPTION
**Pull request by sanketmod**

### CODE REVIEW

File: sanket/javascript/day15/assignment1/sanketA1_Q1.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <script>
+      function btn1() {
```
Comment (KomalShinde813) [inline sanket/javascript/day15/assignment1/sanketA1_Q1.html]: change function name, function name should match to functionality and don't use numbers in function name( e.g swapNumbers() )

File: sanket/javascript/day15/assignment1/sanketA1_Q1.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        let a = parseInt(document.getElementById("num1").value);
+        let b = parseInt(document.getElementById("num2").value);
+        let t = a;
+        a = b;
+        b = t;
+        document.getElementById(
+          "m1"
+        ).innerHTML = `<h3>m1:with temp variable</h3> after swapping A:${a} B:${b}`;
+      }
+      function btn2() {
```
Comment (KomalShinde813) [inline sanket/javascript/day15/assignment1/sanketA1_Q1.html]: change function name and one line space required after one function

File: sanket/javascript/day15/assignment1/sanketA1_Q1.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    <script>
+      function btn1() {
+        let a = parseInt(document.getElementById("num1").value);
+        let b = parseInt(document.getElementById("num2").value);
+        let t = a;
+        a = b;
+        b = t;
+        document.getElementById(
+          "m1"
+        ).innerHTML = `<h3>m1:with temp variable</h3> after swapping A:${a} B:${b}`;
```
Comment (KomalShinde813) [inline sanket/javascript/day15/assignment1/sanketA1_Q1.html]: space required after :

File: sanket/javascript/day15/assignment1/sanketA1_Q1.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      }
+      function btn2() {
+        let a = parseInt(document.getElementById("num1").value);
+        let b = parseInt(document.getElementById("num2").value);
+        b = a + b - (a = b);
+        document.getElementById(
+          "m2"
+        ).innerHTML = `<h3>m2:without temporary variable</h3> after swapping A:${a} B:${b}`;
```
Comment (KomalShinde813) [inline sanket/javascript/day15/assignment1/sanketA1_Q1.html]: space require after :

File: sanket/javascript/day15/assignment1/sanketA1_Q1.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day15/assignment1/sanketA1_Q1.html]: title should be swap two numbers

File: sanket/javascript/day15/assignment1/sanketA1_Q2.html [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day15/assignment1/sanketA1_Q2.html]: title should be program to convert Fahrenite to Celcius

File: sanket/javascript/day15/assignment1/sanketA1_Q2.html [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    <script>
+      function btn() {
```
Comment (KomalShinde813) [inline sanket/javascript/day15/assignment1/sanketA1_Q2.html]: change function name

File: sanket/javascript/day16/assignment2/sanketA2_Q1.html [hunk 8]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day16/assignment2/sanketA2_Q1.html]: change title

File: sanket/javascript/day16/assignment2/sanketA2_Q1.html [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+
+    <script>
+      function cal() {
+        let n1 = parseInt(document.getElementById("no1").value);
+        let n2 = parseInt(document.getElementById("no2").value);
+        let n3 = parseInt(document.getElementById("no3").value);
+        let n4 = parseInt(document.getElementById("no4").value);
+        let n5 = parseInt(document.getElementById("no5").value);
+        // let no = [n1, n2, n3, n4, n5];
+        // let lar = no.sort();
```
Comment (KomalShinde813) [inline sanket/javascript/day16/assignment2/sanketA2_Q1.html]: remove commented code

File: sanket/javascript/day16/assignment2/sanketA2_Q2.html [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+    <script>
+      function cal() {
+        let n1 = parseInt(document.getElementById("no1").value);
+        let n2 = parseInt(document.getElementById("no2").value);
+        let n3 = parseInt(document.getElementById("no3").value);
+        let n4 = parseInt(document.getElementById("no4").value);
+        let n5 = parseInt(document.getElementById("no5").value);
+        let t = n1 + n2 + n3 + n4 + n5;
+        let p = (t / 500) * 100;
+        let g;
```
Comment (KomalShinde813) [inline sanket/javascript/day16/assignment2/sanketA2_Q2.html]: change variable name variable name should be related change everywhere

File: sanket/javascript/day17/assignment3/sanketA3_Q1.html [hunk 11]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day17/assignment3/sanketA3_Q1.html]: change title

File: sanket/javascript/day17/assignment3/sanketA3_Q2.html [hunk 12]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day17/assignment3/sanketA3_Q2.html]: change title

File: sanket/javascript/day17/assignment3/sanketA3_Q2.html [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+  </head>
+  <body>
+    no1: <input type="text" id="no1" />
+    <button onclick="cal()">calculate</button>
+
+    <script>
+      function cal() {
+        let n = parseInt(document.getElementById("no1").value);
```
Comment (KomalShinde813) [inline sanket/javascript/day17/assignment3/sanketA3_Q2.html]: change variable name

File: sanket/javascript/day17/assignment3/sanketA3_Q3.html [hunk 14]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day17/assignment3/sanketA3_Q3.html]: change title

File: sanket/javascript/day17/assignment3/sanketA3_Q5.html [hunk 15]
```diff
@@ -0,0 +1,22 @@
...
+  <body>
+    no1: <input type="text" id="no1" />
+    <button onclick="cal()">calculate</button>
+    <p id="p"></p>
+    <script>
+      function cal() {
+        let n = parseInt(document.getElementById("no1").value);
+        let f = 1;
+        for (i = 1; i <= n; i++) f = f * i;
+        document.getElementById("p").innerHTML = `Factorial  = ${f} `;
```
Comment (KomalShinde813) [inline sanket/javascript/day17/assignment3/sanketA3_Q5.html]: remove extra space

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Comment by sanketmod** [comment]
i know the folder structure is wrong, i am still waiting for  Sourabh  Bhojane sir to help me and thank you sir for pointing it out,  i don't want to messed up it again sorry for the same.

File: sanket/javascript/day15/assignment1/sanketA1_Q1.html (hunk 1 above)
Comment (sanketmod) [reply sanket/javascript/day15/assignment1/sanketA1_Q1.html]: ma'am thank you for you reply i'll change the function name to match the functionally and i'll not use numbers in function name i'll do the required changes asap

File: sanket/javascript/day15/assignment1/sanketA1_Q1.html (hunk 2 above)
Comment (sanketmod) [reply sanket/javascript/day15/assignment1/sanketA1_Q1.html]: ma'am thank you for you reply i'll change the function name   i'll do the required changes asap but i don't understand this "one line space required after one function" can you please help me with the same

File: sanket/javascript/day15/assignment1/sanketA1_Q1.html (hunk 3 above)
Comment (sanketmod) [reply sanket/javascript/day15/assignment1/sanketA1_Q1.html]: ma'am thank you for you reply  don't understand this "space required after : " means what

File: sanket/javascript/day15/assignment1/sanketA1_Q1.html (hunk 4 above)
Comment (sanketmod) [reply sanket/javascript/day15/assignment1/sanketA1_Q1.html]: ma'am thank you for you reply don't understand this "space required after : " means what

File: sanket/javascript/day17/assignment3/sanketA3_Q1.html (hunk 11 above)
Comment (sanketmod) [reply sanket/javascript/day17/assignment3/sanketA3_Q1.html]: ma'am thank you for you reply  i'll change the title asap and please suggest the naming standard for title

File: sanket/javascript/day17/assignment3/sanketA3_Q2.html (hunk 12 above)
Comment (sanketmod) [reply sanket/javascript/day17/assignment3/sanketA3_Q2.html]: ma'am thank you for you reply i'll change the title asap and please suggest the naming standard for title

File: sanket/javascript/day17/assignment3/sanketA3_Q2.html (hunk 13 above)
Comment (sanketmod) [reply sanket/javascript/day17/assignment3/sanketA3_Q2.html]: ma'am thank you for you reply i'll change the variable name can you please specify the variable naming standard you want me to apply

File: sanket/javascript/day17/assignment3/sanketA3_Q3.html (hunk 14 above)
Comment (sanketmod) [reply sanket/javascript/day17/assignment3/sanketA3_Q3.html]: ma'am thank you for you reply i'll change the title asap and please suggest the naming standard for title

File: sanket/javascript/day17/assignment3/sanketA3_Q5.html (hunk 15 above)
Comment (sanketmod) [reply sanket/javascript/day17/assignment3/sanketA3_Q5.html]: ma'am thank you for you reply i'll remove extra space asap if you specify where is the extra space

File: sanket/javascript/day15/assignment1/sanketA1_Q1.html (hunk 5 above)
Comment (sanketmod) [reply sanket/javascript/day15/assignment1/sanketA1_Q1.html]: ma'am thank you for you reply i'll change the title asap

File: sanket/javascript/day15/assignment1/sanketA1_Q2.html (hunk 6 above)
Comment (sanketmod) [reply sanket/javascript/day15/assignment1/sanketA1_Q2.html]: ma'am thank you for you reply i'll change the title asap

File: sanket/javascript/day15/assignment1/sanketA1_Q2.html (hunk 7 above)
Comment (sanketmod) [reply sanket/javascript/day15/assignment1/sanketA1_Q2.html]: ma'am thank you for you reply i'll change the function name asap

File: sanket/javascript/day16/assignment2/sanketA2_Q1.html (hunk 8 above)
Comment (sanketmod) [reply sanket/javascript/day16/assignment2/sanketA2_Q1.html]: ma'am thank you for you reply i'll change the title asap

File: sanket/javascript/day16/assignment2/sanketA2_Q1.html (hunk 9 above)
Comment (sanketmod) [reply sanket/javascript/day16/assignment2/sanketA2_Q1.html]: ma'am thank you for you reply i'll remove the commented code

File: sanket/javascript/day16/assignment2/sanketA2_Q2.html (hunk 10 above)
Comment (sanketmod) [reply sanket/javascript/day16/assignment2/sanketA2_Q2.html]: ma'am thank you for you reply i'll change the variable name can you please specify the variable naming standard you want me to apply

File: sanket/javascript/day15/assignment1/sanketA1_Q1.html (hunk 3 above)
Comment (KomalShinde813) [reply sanket/javascript/day15/assignment1/sanketA1_Q1.html]: `<h3>m1: with temp variable</h3> after swapping A: ${a} B: ${b}`;

File: sanket/javascript/day16/assignment2/sanketA2_Q2.html (hunk 10 above)
Comment (KomalShinde813) [reply sanket/javascript/day16/assignment2/sanketA2_Q2.html]: like var percent, grade, total or total_marks

Comment (KomalShinde813) [reply sanket/javascript/day16/assignment2/sanketA2_Q2.html]: just don't use a, b, n, x, y, z

File: sanket/javascript/day17/assignment3/sanketA3_Q1.html (hunk 11 above)
Comment (KomalShinde813) [reply sanket/javascript/day17/assignment3/sanketA3_Q1.html]: here title should be 'calculating given year is leap year or not'

File: sanket/javascript/day17/assignment3/sanketA3_Q5.html (hunk 15 above)
Comment (KomalShinde813) [reply sanket/javascript/day17/assignment3/sanketA3_Q5.html]: extra space is present before =or after Factorial

File: sanket/javascript/day17/assignment3/sanketA3_Q3.html (hunk 14 above)
Comment (KomalShinde813) [reply sanket/javascript/day17/assignment3/sanketA3_Q3.html]: mention this in above comment change title for all assignments

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Review by VIP5614 — APPROVED** [review]

**Review by MKadeer17 — APPROVED** [review]

File: sanket/javascript/day16/assignment2/sanketA2_Q1.html [hunk 16]
```diff
@@ -4,10 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>
-      Javasript program to read 5 numbers from user and find the second largest
-      number among the list.
-    </title>
+    <title>second largest number</title>
   </head>
```
Comment (VIP5614) [inline sanket/javascript/day16/assignment2/sanketA2_Q1.html]: remove extra line

File: sanket/javascript/day16/assignment2/sanketA2_Q2.html [hunk 17]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>total and percentage and grade</title>
   </head>
```
Comment (VIP5614) [inline sanket/javascript/day16/assignment2/sanketA2_Q2.html]: why add extra line @sanketmod

File: sanket/javascript/day17/assignment3/sanketA3_Q2.html [hunk 18]
```diff
@@ -4,17 +4,17 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>check if it is a prime number or not</title>
   </head>
   <body>
     no1: <input type="text" id="no1" />
     <button onclick="cal()">calculate</button>
```
Comment (VIP5614) [inline sanket/javascript/day17/assignment3/sanketA3_Q2.html]: same issue

**Review by VIP5614 — CHANGES_REQUESTED** [review]

**Review by VIP5614 — APPROVED** [review]

**Review by poonam-khode — APPROVED** [review]

**Review by MKadeer17 — APPROVED** [review]

**Review by pratikshaghule — APPROVED** [review]

File: sanket/javascript/day17/assignment3/sanketA3_Q5.html [hunk 19]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day17/assignment3/sanketA3_Q5.html]: change title

File: sanket/javascript/day18/assignment4/sanketA4_Q1.html [hunk 20]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day18/assignment4/sanketA4_Q1.html]: change title

File: sanket/javascript/day18/assignment4/sanketA4_Q2.html [hunk 21]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day18/assignment4/sanketA4_Q2.html]: change title

File: sanket/javascript/day18/assignment4/sanketA4_Q2.html [hunk 22]
```diff
@@ -0,0 +1,29 @@
...
+  </head>
+  <body>
+    no: <input type="text" id="no" />
+    <button onclick="cal()">calculate</button>
+    <p id="p"></p>
+
+    <script>
+      function cal() {
+        let n = parseInt(document.getElementById("no").value);
+        let r;
```
Comment (KomalShinde813) [inline sanket/javascript/day18/assignment4/sanketA4_Q2.html]: change variable names don't use a, b, c, x, y, z as a variable name give proper relative name to the functionality 
and make this change in each and every assignment

File: sanket/javascript/day18/assignment4/sanketA4_Q3.html [hunk 23]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day18/assignment4/sanketA4_Q3.html]: change title

File: sanket/javascript/day18/assignment4/sanketA4_Q4.html [hunk 24]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day18/assignment4/sanketA4_Q4.html]: change title

File: sanket/javascript/day18/assignment4/sanketA4_Q5.html [hunk 25]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day18/assignment4/sanketA4_Q5.html]: change title

File: sanket/javascript/day19/assignment5/sanketA5_Q1.html [hunk 26]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day19/assignment5/sanketA5_Q1.html]: change title

File: sanket/javascript/day19/assignment5/sanketA5_Q2.html [hunk 27]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day19/assignment5/sanketA5_Q2.html]: change title

File: sanket/javascript/day19/assignment5/sanketA5_Q3.html [hunk 28]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day19/assignment5/sanketA5_Q3.html]: change title

File: sanket/javascript/day19/assignment5/sanketA5_Q4.html [hunk 29]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day19/assignment5/sanketA5_Q4.html]: change title

File: sanket/javascript/day19/assignment5/sanketA5_Q5.html [hunk 30]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day19/assignment5/sanketA5_Q5.html]: change title

File: sanket/javascript/day20/assignment6/sanketA6_Q1.html [hunk 31]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day20/assignment6/sanketA6_Q1.html]: change title

File: sanket/javascript/day20/assignment6/sanketA6_Q2.html [hunk 32]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day20/assignment6/sanketA6_Q2.html]: change title

File: sanket/javascript/day20/assignment6/sanketA6_Q3.html [hunk 33]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day20/assignment6/sanketA6_Q3.html]: change title

File: sanket/javascript/day20/assignment6/sanketA6_Q4.html [hunk 34]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day20/assignment6/sanketA6_Q4.html]: change title

File: sanket/javascript/day20/assignment6/sanketA6_Q5.html [hunk 35]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline sanket/javascript/day20/assignment6/sanketA6_Q5.html]: change the title

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Review by KomalShinde813 — APPROVED** [review]

**Pull request merged**